library/Icinga/Web/Navigation/Mobile/MobileConfigMenu.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/* Icinga Web 2 | (c) 2022 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Web\Navigation\Mobile;
+
+use Icinga\Application\Icinga;
+use Icinga\Authentication\Auth;
+use Icinga\Common\HealthBadgeTrait;
+use Icinga\Web\Navigation\ConfigMenu;
+use ipl\Html\Attributes;
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\Web\Url;
+use ipl\Web\Widget\Icon;
+
+class MobileConfigMenu extends ConfigMenu
+{
+    use HealthBadgeTrait;
+
+    protected $tag = 'ul';
+
+    protected $defaultAttributes = [
+        'class' => 'nav user-nav-item',
+        'id' => 'mobile-config-menu'
+    ];
+
+    protected $flyoutID = 'mobile-config-menu-flyout';
+
+    protected function isSelectedItem($item)
+    {
+        if ($item !== null && Icinga::app()->getRequest()->getUrl()->matches($item['url'])) {
+            $this->selected = $item;
+            return true;
+        }
+
+        return false;
+    }
+
+    protected function assembleLevel2Nav(BaseHtmlElement $level2Nav)
+    {
+        $username = Auth::getInstance()->getUser()->getUsername();
+        $navContent = HtmlElement::create('div', ['class' => 'flyout-content']);
+        $navContent->add(HtmlElement::create('ul', ['class' => 'nav flyout-menu'], [
+            HtmlElement::create('li', ['class' => 'has-icon'], [
+                HtmlElement::create(
+                    'a',
+                    Attributes::create(['href' => Url::fromPath('account')]),
+                    [
+                        HtmlElement::create('i', ['class' => 'user-ball'], Text::create($username[0])),
+                        Text::create($username)
+                    ]
+                )
+            ])
+        ]));
```

Comment:
Please reduce it to this block + the user name retrieval by first calling the base implementation and then by accessing the `.flyout-content` with `$level2Nav->getFirst('div')`.

Reply:
✅